kornia/contrib/models/tiny_vit.py

```diff
@@ -85,7 +85,10 @@ def __init__(
 
     def forward(self, x: Tensor) -> Tensor:
         if x.ndim == 3:
-            x = x.transpose(1, 2).unflatten(2, self.input_resolution)  # (B, H * W, C) -> (B, C, H, W)
+            # x = x.transpose(1, 2).unflatten(2, self.input_resolution)  # (B, H * W, C) -> (B, C, H, W)
```

Comment:
@ducha-aiki i noticed that some models for matching use unflatten too which is not supported by onnx